client_functions/listener.py

```diff
@@ -1,6 +1,19 @@
+#Document information
+document_data = {
```

Comment:
Not sure about this approach of adding the header to every page, might be more confusing. 
Github would keep the versioning for every page. 
Not as to remove on this PR, but then can be done a PR as to remove from all pages.